phpstan-v66-baseline.neon

```diff
@@ -650,6 +650,11 @@ parameters:
             count: 3
             path: src/Core/Framework/Api/Controller/IndexingController.php
 
+        -
+            message: "#^Throwing new exceptions within classes are not allowed\\. Please use domain exception pattern\\. See https\\://github\\.com/shopware/platform/blob/v6\\.4\\.20\\.0/adr/2022\\-02\\-24\\-domain\\-exceptions\\.md$#"
```

Comment:
can you solve this, instead of adding new entries? 🤔

Reply:
That entry is here because of the `InvalidContextSourceException` & `ExpectedUserHttpException` in `SalesChannelProxyController`. There are currently no static functions to create this Exceptions. This is also the case in every other file using this exceptions like https://github.com/shopware/shopware/blob/trunk/src/Core/Framework/Api/Controller/UserController.php#L53

Reply:
Ahh okay.